README.md

```diff
@@ -12,69 +12,73 @@ This specification defines the additional OpenID Connect authentication response
 - Adam Lemmon (Trybe.ID)
 - Tobias Looker (Mattr)
 
+## Terminology
+
+If the specification states "Verifiable Presentation" this also includes "Verifiable Credentials". 
 ## Introduction
 
 Notes:
 - this should really start with the explanation of the role signatures play in VCs (can be both JSON or JSON-LD) and that there are two widely used proof types (JWTs and LD-proofs)
 - Explain why is there a need for this extension?
 
 ## Overview
-- RP requests `vp_token` by adding an additional destination `vp_token` in the `claims` request parameter. 
-- Authentication event information is conveyed via the id token while it's up to the RP to determine what (additional) claims are allocated to id_token and vp_token, respectively.
-- `vp_token` is provided in the same response as the `id_token`. Depending on the response type, this can be either the authentication response or the token response. 
-- If the vp_token is returned in the frontchannel, a hash (`vp_hash`) of `vp_token` must be included in `id_token`.
+
+This specifications introduces two mechanisms to provide Verifiable Presentations:
+
+* ID Token as Verififiable Presentation: An ID Token may contain a claim `vp` or `vc` as defined in [JWT proof format](https://www.w3.org/TR/vc-data-model/#json-web-token), i.e. it is a valid OpenID Connect ID Token and a Verifiable Presentation at the same time. Consequently, this mechanism utilizes (and supports) the JWT proof format only. 
```

Comment:
Do we want to support both `vp` and `vc`?
`vc` would generally be what the Holder (OP) has acquired during a credential issuance flow from an issuer and thus signed by that issuer.

So for the Holder to present that to the RP here they would need to wrap it with the `vp` envelope I think? Without then there would be no proof of control over the `vc`.  I suppose the Holder could present self asserted claims in the form of a `vc` if they wanted though... where they are both the issuer and subject.. but I don't know if that was the intended use here?

Reply:
Very good point! I don’t think a VC alone is not useful, 
@awoie @Sakurann WDYT?

Reply:
I would agreed :)
ping @awoie @Sakurann

Reply:
we discussed this point and came to the conclusion there are use cases where a VC alone can be provided, e.g. if the VC is a bearer credential. I updated the text accordingly.

Reply:
@tlodderstedt I think we discussed an example of a bearer VC inside a VP signed by a holder. I think for OIDC, supporting VP is enough for the use-cases I know now. also see [https://bitbucket.org/openid/connect/issues/1205/indicating-support-for-vcs-siop#comment-60170872](https://bitbucket.org/openid/connect/issues/1205/indicating-support-for-vcs-siop#comment-60170872)

Reply:
to me, just using vp is fine.